rpcs3/Emu/Cell/Modules/cellMic.cpp

```diff
@@ -95,6 +95,12 @@ void mic_context::operator()()
 		{
 			std::lock_guard lock(mutex);
 
+			if (mic_list.empty())
+			{
+				thread_ctrl::wait_for(100000);
```

Comment:
We shouldn't do this stall while holding the lock.

Reply:
We already know the mic list is empty at the start of the loop. Just add it to the pause condition so that it short-circuits `if (Emu.IsPaused() || mic_list.empty())`